ihatemoney/templates/layout.html

```diff
@@ -12,6 +12,7 @@
     <script src="{{ url_for("static", filename="js/popper.min.js") }}"></script>
     <script src="{{ url_for("static", filename="js/tagsinput.js") }}"></script>
     <script src="{{ url_for("static", filename="js/bootstrap.min.js") }}"></script>
+    <script src="{{ url_for("static", filename="js/moment.min.js") }}"></script>
```

Comment:
momentjs is really heavy, is it's really necessarry? 
If i'm correct, it's just to format calendar... shouldn't we try to use smth slimer or serverside?
or couldn't we fallback to simpler date format

Reply:
momentjs is a really nice library, but I think using it here is a bit overkill.
We could simply display ISO dates (or a bit simplified), since the history page doesn't need to be user friendly.

Reply:
We can also use python to format the dates. Something like arrow (with this humanize() fonction)

Reply:
Will the server-side solutions correct for time zone? That was the reason I thought a JavaScript library would be necessary

Reply:
We can add a small hand-made javascript that just do:
```javascript
function (utcTimestamp) {
    return new Date(utcTimestamp).toISOString()
}
```

We could just have a simple format instead of `toISOString`. I know that this is exactly the goal of `moment`, but I still think that we should keep away from it for now.

Reply:
We can actually still do some localization, [Date.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) is supported by all modern browsers. Implemented as of ddd2600f